src/vorta/application.py

```diff
@@ -1,25 +1,26 @@
 import logging
 import os
 import sys
+from typing import Any, Dict
 
 from PyQt5 import QtCore
 from PyQt5.QtWidgets import QMessageBox
 
+from vorta.borg.break_lock import BorgBreakJob
 from vorta.borg.create import BorgCreateJob
+from vorta.borg.jobs_manager import JobsManager
 from vorta.borg.version import BorgVersionJob
-from vorta.borg.break_lock import BorgBreakJob
-from vorta.config import TEMP_DIR, PROFILE_BOOTSTRAP_FILE
+from vorta.config import PROFILE_BOOTSTRAP_FILE, TEMP_DIR
 from vorta.i18n import init_translations, translate
-from vorta.store.models import BackupProfileModel, SettingsModel
-from vorta.store.connection import cleanup_db
+from vorta.notifications import VortaNotifications
+from vorta.profile_export import ProfileExport
 from vorta.qt_single_application import QtSingleApplication
 from vorta.scheduler import VortaScheduler
-from vorta.borg.jobs_manager import JobsManager
+from vorta.store.connection import cleanup_db
+from vorta.store.models import BackupProfileModel, SettingsModel
 from vorta.tray_menu import TrayMenu
 from vorta.utils import borg_compat, parse_args
 from vorta.views.main_window import MainWindow
```

Comment:
My IDE is using `isort` to sort imports. I didn't notice this found its way into a commit. Sorry about that.

Reply:
No worries. I plan to use Black and isort to clean up the whole code base at some point. It's just very destructive to the history and if distros want to cherry-pick only certain commits.